src/app/zap-templates/zcl/data-model/chip/general-commissioning-cluster.xml

```diff
@@ -50,6 +63,26 @@ limitations under the License.
     <attribute side="server" code="0x02" define="REGULATORY_CONFIG" type="RegulatoryLocationTypeEnum" writable="false" optional="false">RegulatoryConfig</attribute>
     <attribute side="server" code="0x03" define="LOCATION_CAPABILITY" type="RegulatoryLocationTypeEnum" writable="false" optional="false">LocationCapability</attribute>
     <attribute side="server" code="0x04" define="SUPPORTS_CONCURRENT_CONNECTION" type="boolean" writable="false" default="1" optional="false">SupportsConcurrentConnection</attribute>
+    <attribute side="server" code="0x05" define="TC_ACCEPTED_VERSION" type="int16u" min="0x0000" max="0xFFFF" writable="false" optional="true" isNullable="true">
```

Comment:
This XML does not match the spec (at least not the spec I see).  Spec does not have any of these attributes nullable.

Reply:
+1 please fix

Reply:
Removed nullable.